Task_DEV_2/Task_DEV_2/Program.cs

```diff
@@ -4,47 +4,64 @@ namespace Task_DEV_2
-
-            if (arrayLength <= 20)
+            int[] inputNumbers = new int[arrayLength];
+            for (int i = 0; i < arrayLength; i++)
             {
-                int[] inputNumbers = new int[arrayLength];
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+            return inputNumbers;
+        }
 
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
-                }
-                int min = inputNumbers[0];
-                int max = inputNumbers[0];
-                int indexMin = 0;
-                int indexMax = 0;
+        static int SwapMinMaxElements(int[] inputNumbers, int arrayLength)
+        {
+            int min = inputNumbers[0];
+            int max = inputNumbers[0];
+            int indexMin = 0;
+            int indexMax = 0;
 
-                for (int i = 0; i < arrayLength; i++)
+            for (int i = 0; i < arrayLength; i++)
+            {
+                if (inputNumbers[i] < min)
                 {
-                    if (inputNumbers[i] < min)
-                    {
-                        min = inputNumbers[i];
-                        indexMin = i;
-                    }
-                    if (inputNumbers[i] > max)
-                    {
-                        max = inputNumbers[i];
-                        indexMax = i;
-                    }
-                    Console.WriteLine(inputNumbers[i]);
+                    min = inputNumbers[i];
+                    indexMin = i;
                 }
-                inputNumbers[indexMin] = max;
-                inputNumbers[indexMax] = min;
-
-                for (int i = 0; i < arrayLength; i++)
+                if (inputNumbers[i] > max)
                 {
-                    Console.WriteLine(inputNumbers[i]);
+                    max = inputNumbers[i];
+                    indexMax = i;
                 }
             }
+            inputNumbers[indexMin] = max;
+            inputNumbers[indexMax] = min;
+
+            return inputNumbers[indexMin];
+        }
+
+        static void PrintNumberArraryToConsole(int[] inputNumbers, int arrayLength)
+        {
+
+            for (int i = 0; i < arrayLength; i++)
+            {
+                Console.WriteLine(inputNumbers[i]);
+            }
+        }
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the size of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 20)
+            {
+                int[] inputNumbers = CreateNumbersArray(arrayLength);
+                PrintNumberArraryToConsole(inputNumbers, arrayLength);
+                SwapMinMaxElements(inputNumbers, arrayLength);
```

Comment:
After swapping would be more logical do smth like this
`inputNumbers = SwapMinMaxElements(inputNumbers, arrayLength);`

Reply:
Done.